mail/mailer.go

```diff
@@ -212,32 +216,40 @@ func (m *MailerImpl) SendMail(to []string, subject, msg string) error {
 	if m.client == nil {
 		return errors.New("call Connect before SendMail")
 	}
+	m.stats.Inc("SendMail.Attempts", 1)
 	body, err := m.generateMessage(to, subject, msg)
 	if err != nil {
+		m.stats.Inc("SendMail.Errors", 1)
```

Comment:
Good idea - I was trying to think of a way to collapse these down with a `goto` but it seemed unpleasant. I'll pull out a function and deal with it that way. Thanks!